docs-ref-conceptual/azure-cli-variables.md

```diff
@@ -44,6 +47,22 @@ osType=$(az vm get-instance-view --resource-group $MyResourceGroup \
 # Create a disk of the same type by using the variable value
 az disk create --resource-group $MyResourceGroup --name DestinationDisk --size-gb 20 --os-type $osType
 ```
+# [PowerShell](#tab/powershell)
+
+```azurecli
+# PowerShell script
+# Assign values to variables
+$MyResourceGroup = "ContosoRGforVM"
+$MySubscription = "Contoso subscription"
+$vmName = "VM01"
+
+# Get a value for a variable based on an existing virtual machine
+$osType = (Get-AzVM -ResourceGroupName $MyResourceGroup -Name $vmName -SubscriptionName $MySubscription).StorageProfile.OsDisk.OsType
+
+# Create a disk of the same type by using the variable value
+New-AzDisk -ResourceGroupName $MyResourceGroup -DiskName DestinationDisk -DiskSizeGB 20 -OsType $osType
```

Comment:
```suggestion
az disk create --resource-group $MyResourceGroup --name DestinationDisk --size-gb 20 --os-type $osType
```

Reply:
@v-abdriaz , The point of this update is to show the customer how to run Azure CLI commands in a PowerShell environment. We should not be replacing Azure CLI commands with Azure PowerShell commands.  We need to make the Azure CLI command work.